### DISCUSSION TITLE
Add performance benchmark

### DISCUSSION THREAD

**Discussion by jellydn** [NvChad/NvChad#1712] [Ideas]
# What
- Add performance benchmark 

# Why
-  This will bring more people to NVChad :)

# How
For example, we could have a similar approach like
[ecosse3/nvim: A non-minimal Neovim config built to work most efficiently with Frontend Development - React & Vue.js](https://github.com/ecosse3/nvim#performance)

**Comment by siduck** [comment]
hi @jellydn sounds good! i will add it in next release which is going to be within few weeks
